### PR TITLE
provider/google: return L7 LB policy from metadata.

### PR DESCRIPTION
**Pull request by jtk54**
@duftler or @lwander or @danielpeach PTAL.

### CODE REVIEW

**Comment by jtk54** [comment]
image

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy [hunk 1]
```diff
@@ -27,13 +27,11 @@ import com.netflix.frigga.Names
 import com.netflix.spectator.api.Registry
 import com.netflix.spinnaker.cats.agent.AgentDataType
 import com.netflix.spinnaker.cats.agent.CacheResult
-import com.netflix.spinnaker.cats.agent.DefaultCacheResult
 import com.netflix.spinnaker.cats.cache.CacheData
```
Comment (duftler) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy]: Sort-order is off here.

**Comment by duftler** [comment]
LGTM.

File: clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy (hunk 1 above)
Comment (jtk54) [inline clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/agent/GoogleRegionalServerGroupCachingAgent.groovy]: `agent.DefaultCacheResult` < `cache.CacheData` here I think, right? Optimizing imports in IntelliJ didn't do anything.

**Pull request merged**